metricbeat/module/munin/node/_meta/data.json

```diff
@@ -0,0 +1,35 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "event": {
+        "dataset": "munin.node",
```

Comment:
@jsoriano I'm thinking if the data set here should actually be `munin.metrics`. Same would apply to prometheus. WDYT?

Reply:
What I like of `munin.node` is that it refers to being a metricset that collects metrics from a munin node. But I don't have an strong opinion, feel free to change it, now that there are going to be so many breaking changes it would be the moment.

Reply:
To be honest I'm on the fence here. I like the name of the metricset itself and would keep it. I'm only thinking about the dataset name.